components/form/Toggle.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import classNames from 'classnames';
+
+const Toggle = ({ containerClassName, label, name, value, toggleClassName }) => {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={classNames('relative flex items-center', containerClassName)}>
+        <input
+          name={name}
+          type="checkbox"
+          {...register(name)}
+          id={`toggle-${name}-${value.toString()}`}
+          value={value}
```

Comment:
We don't need to set a value with Toggle as Toggle is meant to be true or false.
```suggestion
          value={true}
```